[INFO ] Logging to: /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/outdir_train_configuration/RandomForest_Iris_scenario/log-run10.txt
[INFO ] Version of SMAC is v2.10.03-master-778 (3ee628ef9bf2), running on OpenJDK 64-Bit Server VM (1.8.0_442) and Linux 4.18.0-553.36.1.el8_10.x86_64 (amd64) 
[INFO ] Call String: smac --scenario-file Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/RandomForest_Iris_scenario.txt --seed 10 
[INFO ] SMAC started at: Apr 14, 2025 8:16:32 AM. Minimizing mean quality.
[INFO ] First incumbent: config 1 (internal ID: 0x000B), with mean quality: -1.0; estimate based on 1 runs.
[INFO ] Sample call for new incumbent config 1 (internal ID: 0x000B): 
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 1399991 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:16:53_54831_1633817259.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:16:53_54831_1633817259.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:16:53_54831_1633817259.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth": "8", "max_depth_type": "int", "max_features_float": "0.044948619508610066", "max_features_type": "float", "max_leaf_nodes": "677", "max_leaf_nodes_type": "int", "max_samples_type": "None", "min_impurity_decrease": "0.07984465361848775", "min_samples_leaf": "89", "min_samples_split": "32", "min_weight_fraction_leaf": "0.47693033557010683", "n_estimators": "8", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "1399991", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 1399991 -bootstrap 'True' -criterion 'log_loss' -max_depth '8' -max_depth_type 'int' -max_features_float '0.044948619508610066' -max_features_type 'float' -max_leaf_nodes '677' -max_leaf_nodes_type 'int' -max_samples_type 'None' -min_impurity_decrease '0.07984465361848775' -min_samples_leaf '89' -min_samples_split '32' -min_weight_fraction_leaf '0.47693033557010683' -n_estimators '8' -oob_score 'False'  
[WARN ] Detected CRASHED run. The Solution Quality of CRASHED runs will be transformed to the MAX(quality, 1.0E9), to disable this use --transform-crashed-quality false to change the value use --transform-crashed-quality-value
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:08_55646_4272541685.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:08_55646_4272541685.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:08_55646_4272541685.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.34063688635983913", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.31932061133667433", "min_samples_leaf": "81", "min_samples_split": "47", "min_weight_fraction_leaf": "0.10859210052778528", "n_estimators": "473", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "3317383", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 3317383 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.34063688635983913' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.31932061133667433' -min_samples_leaf '81' -min_samples_split '47' -min_weight_fraction_leaf '0.10859210052778528' -n_estimators '473'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:23_56307_2891255504.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:23_56307_2891255504.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:23_56307_2891255504.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth": "6", "max_depth_type": "int", "max_features_float": "0.9501934093075491", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples_type": "None", "min_impurity_decrease": "0.3281228977918155", "min_samples_leaf": "52", "min_samples_split": "66", "min_weight_fraction_leaf": "0.1782715204150258", "n_estimators": "114", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "5878414", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 5878414 -bootstrap 'True' -criterion 'gini' -max_depth '6' -max_depth_type 'int' -max_features_float '0.9501934093075491' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.3281228977918155' -min_samples_leaf '52' -min_samples_split '66' -min_weight_fraction_leaf '0.1782715204150258' -n_estimators '114' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:38_56768_2699460639.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:38_56768_2699460639.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:38_56768_2699460639.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth_type": "None", "max_features_special": "log2", "max_features_type": "special", "max_leaf_nodes": "747", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.10290815905565431", "min_samples_leaf": "61", "min_samples_split": "82", "min_weight_fraction_leaf": "0.21207473521231446", "n_estimators": "246", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12348508", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12348508 -bootstrap 'False' -criterion 'log_loss' -max_depth_type 'None' -max_features_special 'log2' -max_features_type 'special' -max_leaf_nodes '747' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.10290815905565431' -min_samples_leaf '61' -min_samples_split '82' -min_weight_fraction_leaf '0.21207473521231446' -n_estimators '246'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:53_57306_898002978.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:53_57306_898002978.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:17:53_57306_898002978.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "entropy", "max_depth_type": "None", "max_features_float": "0.2745788356726514", "max_features_type": "float", "max_leaf_nodes": "223", "max_leaf_nodes_type": "int", "max_samples_type": "None", "min_impurity_decrease": "0.25321755579518795", "min_samples_leaf": "30", "min_samples_split": "73", "min_weight_fraction_leaf": "0.06586400067281406", "n_estimators": "404", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "9627338", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 9627338 -bootstrap 'True' -criterion 'entropy' -max_depth_type 'None' -max_features_float '0.2745788356726514' -max_features_type 'float' -max_leaf_nodes '223' -max_leaf_nodes_type 'int' -max_samples_type 'None' -min_impurity_decrease '0.25321755579518795' -min_samples_leaf '30' -min_samples_split '73' -min_weight_fraction_leaf '0.06586400067281406' -n_estimators '404' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:08_58177_2792362977.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:08_58177_2792362977.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:08_58177_2792362977.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth_type": "None", "max_features_special": "log2", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.22707153640248934", "min_samples_leaf": "35", "min_samples_split": "49", "min_weight_fraction_leaf": "0.12197262701979988", "n_estimators": "67", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "9627338", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 9627338 -bootstrap 'False' -criterion 'entropy' -max_depth_type 'None' -max_features_special 'log2' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.22707153640248934' -min_samples_leaf '35' -min_samples_split '49' -min_weight_fraction_leaf '0.12197262701979988' -n_estimators '67'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:23_58782_2682803513.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:23_58782_2682803513.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:23_58782_2682803513.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth": "4", "max_depth_type": "int", "max_features_special": "log2", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.26272490891337724", "min_samples_leaf": "7", "min_samples_split": "77", "min_weight_fraction_leaf": "0.005112413572746333", "n_estimators": "404", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "9480515", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 9480515 -bootstrap 'False' -criterion 'log_loss' -max_depth '4' -max_depth_type 'int' -max_features_special 'log2' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.26272490891337724' -min_samples_leaf '7' -min_samples_split '77' -min_weight_fraction_leaf '0.005112413572746333' -n_estimators '404'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:38_59249_2132117260.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:38_59249_2132117260.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:18:38_59249_2132117260.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth": "43", "max_depth_type": "int", "max_features_float": "0.7843133080235916", "max_features_type": "float", "max_leaf_nodes": "799", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.24387216674573103", "min_samples_leaf": "92", "min_samples_split": "66", "min_weight_fraction_leaf": "0.45160274834631586", "n_estimators": "374", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "3317383", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 3317383 -bootstrap 'False' -criterion 'entropy' -max_depth '43' -max_depth_type 'int' -max_features_float '0.7843133080235916' -max_features_type 'float' -max_leaf_nodes '799' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.24387216674573103' -min_samples_leaf '92' -min_samples_split '66' -min_weight_fraction_leaf '0.45160274834631586' -n_estimators '374'  
[INFO ] Updated estimated mean quality of the same incumbent: -0.9633333333333334; estimate now based on 10 runs.
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:08_60710_938004139.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:08_60710_938004139.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:19:08_60710_938004139.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.49486071525856823", "max_features_type": "float", "max_leaf_nodes": "473", "max_leaf_nodes_type": "int", "max_samples": "0.54823094632656", "max_samples_type": "float", "min_impurity_decrease": "0.46001755809356276", "min_samples_leaf": "88", "min_samples_split": "59", "min_weight_fraction_leaf": "0.2214541556528391", "n_estimators": "150", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "1399991", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 1399991 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.49486071525856823' -max_features_type 'float' -max_leaf_nodes '473' -max_leaf_nodes_type 'int' -max_samples '0.54823094632656' -max_samples_type 'float' -min_impurity_decrease '0.46001755809356276' -min_samples_leaf '88' -min_samples_split '59' -min_weight_fraction_leaf '0.2214541556528391' -n_estimators '150' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:24_63883_2109747864.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:24_63883_2109747864.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:24_63883_2109747864.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth": "50", "max_depth_type": "int", "max_features_float": "0.04737596573435454", "max_features_type": "float", "max_leaf_nodes": "260", "max_leaf_nodes_type": "int", "max_samples": "0.7165450523764911", "max_samples_type": "float", "min_impurity_decrease": "0.11312162404746195", "min_samples_leaf": "95", "min_samples_split": "73", "min_weight_fraction_leaf": "0.17503213486299596", "n_estimators": "139", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12348508", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12348508 -bootstrap 'True' -criterion 'log_loss' -max_depth '50' -max_depth_type 'int' -max_features_float '0.04737596573435454' -max_features_type 'float' -max_leaf_nodes '260' -max_leaf_nodes_type 'int' -max_samples '0.7165450523764911' -max_samples_type 'float' -min_impurity_decrease '0.11312162404746195' -min_samples_leaf '95' -min_samples_split '73' -min_weight_fraction_leaf '0.17503213486299596' -n_estimators '139' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:39_64484_2973578532.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:39_64484_2973578532.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:39_64484_2973578532.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "entropy", "max_depth": "30", "max_depth_type": "int", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes_type": "None", "max_samples_type": "None", "min_impurity_decrease": "0.31062123598949853", "min_samples_leaf": "3", "min_samples_split": "32", "min_weight_fraction_leaf": "1.6015261542096093E-4", "n_estimators": "449", "oob_score": "True", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "1399991", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 1399991 -bootstrap 'True' -criterion 'entropy' -max_depth '30' -max_depth_type 'int' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.31062123598949853' -min_samples_leaf '3' -min_samples_split '32' -min_weight_fraction_leaf '1.6015261542096093E-4' -n_estimators '449' -oob_score 'True'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:54_65060_280856073.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:54_65060_280856073.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:20:54_65060_280856073.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_special": "log2", "max_features_type": "special", "max_leaf_nodes": "408", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.48072381717486845", "min_samples_leaf": "77", "min_samples_split": "30", "min_weight_fraction_leaf": "0.29090763261359454", "n_estimators": "133", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "4879484", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 4879484 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_special 'log2' -max_features_type 'special' -max_leaf_nodes '408' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.48072381717486845' -min_samples_leaf '77' -min_samples_split '30' -min_weight_fraction_leaf '0.29090763261359454' -n_estimators '133'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:09_68181_3968073922.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:09_68181_3968073922.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:09_68181_3968073922.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth": "4", "max_depth_type": "int", "max_features_special": "sqrt", "max_features_type": "special", "max_leaf_nodes": "355", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.3869936949929205", "min_samples_leaf": "8", "min_samples_split": "6", "min_weight_fraction_leaf": "0.14214488444944812", "n_estimators": "8", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "957168", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 957168 -bootstrap 'False' -criterion 'log_loss' -max_depth '4' -max_depth_type 'int' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes '355' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.3869936949929205' -min_samples_leaf '8' -min_samples_split '6' -min_weight_fraction_leaf '0.14214488444944812' -n_estimators '8'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:24_68800_4140232361.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:24_68800_4140232361.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:24_68800_4140232361.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "log_loss", "max_depth": "92", "max_depth_type": "int", "max_features_float": "0.4616061208841744", "max_features_type": "float", "max_leaf_nodes_type": "None", "max_samples": "0.652425175184789", "max_samples_type": "float", "min_impurity_decrease": "0.06025687945471725", "min_samples_leaf": "1", "min_samples_split": "21", "min_weight_fraction_leaf": "0.4244250005507616", "n_estimators": "325", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "9480515", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 9480515 -bootstrap 'True' -criterion 'log_loss' -max_depth '92' -max_depth_type 'int' -max_features_float '0.4616061208841744' -max_features_type 'float' -max_leaf_nodes_type 'None' -max_samples '0.652425175184789' -max_samples_type 'float' -min_impurity_decrease '0.06025687945471725' -min_samples_leaf '1' -min_samples_split '21' -min_weight_fraction_leaf '0.4244250005507616' -n_estimators '325' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:39_69324_2579267317.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:39_69324_2579267317.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:39_69324_2579267317.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.3624411114541062", "max_features_type": "float", "max_leaf_nodes": "676", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.3208708435025789", "min_samples_leaf": "95", "min_samples_split": "71", "min_weight_fraction_leaf": "0.08493740901660485", "n_estimators": "368", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "1399991", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 1399991 -bootstrap 'False' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.3624411114541062' -max_features_type 'float' -max_leaf_nodes '676' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.3208708435025789' -min_samples_leaf '95' -min_samples_split '71' -min_weight_fraction_leaf '0.08493740901660485' -n_estimators '368'  
[INFO ] Updated estimated mean quality of the same incumbent: -0.965; estimate now based on 20 runs.
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:53_69814_2927495275.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:53_69814_2927495275.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:22:53_69814_2927495275.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth": "49", "max_depth_type": "int", "max_features_float": "0.7908485159596726", "max_features_type": "float", "max_leaf_nodes": "539", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.006385621490645865", "min_samples_leaf": "71", "min_samples_split": "6", "min_weight_fraction_leaf": "0.010739141709534328", "n_estimators": "45", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "4937403", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 4937403 -bootstrap 'False' -criterion 'entropy' -max_depth '49' -max_depth_type 'int' -max_features_float '0.7908485159596726' -max_features_type 'float' -max_leaf_nodes '539' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.006385621490645865' -min_samples_leaf '71' -min_samples_split '6' -min_weight_fraction_leaf '0.010739141709534328' -n_estimators '45'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:08_70489_287821863.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:08_70489_287821863.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:23:08_70489_287821863.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth": "33", "max_depth_type": "int", "max_features_special": "log2", "max_features_type": "special", "max_leaf_nodes": "900", "max_leaf_nodes_type": "int", "min_impurity_decrease": "0.28061595887640683", "min_samples_leaf": "25", "min_samples_split": "96", "min_weight_fraction_leaf": "0.4511511859950459", "n_estimators": "491", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "1061282", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 1061282 -bootstrap 'False' -criterion 'log_loss' -max_depth '33' -max_depth_type 'int' -max_features_special 'log2' -max_features_type 'special' -max_leaf_nodes '900' -max_leaf_nodes_type 'int' -min_impurity_decrease '0.28061595887640683' -min_samples_leaf '25' -min_samples_split '96' -min_weight_fraction_leaf '0.4511511859950459' -n_estimators '491'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:18_76826_55123903.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:18_76826_55123903.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:18_76826_55123903.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "entropy", "max_depth_type": "None", "max_features_special": "None", "max_features_type": "special", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.20972188471867886", "min_samples_leaf": "22", "min_samples_split": "82", "min_weight_fraction_leaf": "0.35966203673801406", "n_estimators": "71", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12645757", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12645757 -bootstrap 'False' -criterion 'entropy' -max_depth_type 'None' -max_features_special 'None' -max_features_type 'special' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.20972188471867886' -min_samples_leaf '22' -min_samples_split '82' -min_weight_fraction_leaf '0.35966203673801406' -n_estimators '71'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:32_77123_576312058.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:32_77123_576312058.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:32_77123_576312058.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "True", "criterion": "gini", "max_depth_type": "None", "max_features_float": "0.2360607621031644", "max_features_type": "float", "max_leaf_nodes": "298", "max_leaf_nodes_type": "int", "max_samples": "0.6226833287744298", "max_samples_type": "float", "min_impurity_decrease": "6.611143350190862E-4", "min_samples_leaf": "95", "min_samples_split": "3", "min_weight_fraction_leaf": "0.006181982079314363", "n_estimators": "116", "oob_score": "False", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "2783606", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 2783606 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_float '0.2360607621031644' -max_features_type 'float' -max_leaf_nodes '298' -max_leaf_nodes_type 'int' -max_samples '0.6226833287744298' -max_samples_type 'float' -min_impurity_decrease '6.611143350190862E-4' -min_samples_leaf '95' -min_samples_split '3' -min_weight_fraction_leaf '0.006181982079314363' -n_estimators '116' -oob_score 'False'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:46_77441_2079193354.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:46_77441_2079193354.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:25:46_77441_2079193354.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "log_loss", "max_depth": "47", "max_depth_type": "int", "max_features_float": "0.8681908782383113", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.41921928843211476", "min_samples_leaf": "78", "min_samples_split": "71", "min_weight_fraction_leaf": "0.4415849392663617", "n_estimators": "255", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "12348508", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 12348508 -bootstrap 'False' -criterion 'log_loss' -max_depth '47' -max_depth_type 'int' -max_features_float '0.8681908782383113' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.41921928843211476' -min_samples_leaf '78' -min_samples_split '71' -min_weight_fraction_leaf '0.4415849392663617' -n_estimators '255'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:15_78228_3445654379.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:15_78228_3445654379.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:15_78228_3445654379.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth": "30", "max_depth_type": "int", "max_features_float": "0.9877321184479698", "max_features_type": "float", "max_leaf_nodes_type": "None", "min_impurity_decrease": "0.48428063077920946", "min_samples_leaf": "5", "min_samples_split": "73", "min_weight_fraction_leaf": "0.05721419715158654", "n_estimators": "399", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "9627338", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 9627338 -bootstrap 'False' -criterion 'gini' -max_depth '30' -max_depth_type 'int' -max_features_float '0.9877321184479698' -max_features_type 'float' -max_leaf_nodes_type 'None' -min_impurity_decrease '0.48428063077920946' -min_samples_leaf '5' -min_samples_split '73' -min_weight_fraction_leaf '0.05721419715158654' -n_estimators '399'  
[WARN ] [PROCESS-ERR]  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/tools/runsolver.py:184: RuntimeWarning: Solver output decoding failed from RunSolver configuration: '/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest/runsolver --timestamp --use-pty --cpu-limit 60.0 -w Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:30_78616_1035162003.log -v Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:30_78616_1035162003.val -o Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp/Iris.csv_RandomForest_2025-04-14-09:26:30_78616_1035162003.rawres Solvers/RandomForest/sparkle_solver_wrapper.py '{"bootstrap": "False", "criterion": "gini", "max_depth": "47", "max_depth_type": "int", "max_features_special": "log2", "max_features_type": "special", "max_leaf_nodes": "869", "max_leaf_nodes_type": "int", "min_impurity_decrease": "1.7492582110184562E-4", "min_samples_leaf": "52", "min_samples_split": "32", "min_weight_fraction_leaf": "0.10583492471874945", "n_estimators": "324", "solver_dir": "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Solvers/RandomForest", "instance": "Instances/Iris/Iris.csv", "seed": "13086944", "objectives": "accuracy:max", "cutoff_time": "60.0"}''. Setting status to 'UNKNOWN'.
[WARN ] [PROCESS-ERR]    warnings.warn("Solver output decoding failed from RunSolver configuration: "
[WARN ] [PROCESS-ERR]  Traceback (most recent call last):
[WARN ] [PROCESS-ERR]    File "/rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py", line 46, in <module>
[WARN ] [PROCESS-ERR]      quality = float(output[objective.name])
[WARN ] [PROCESS-ERR]  TypeError: float() argument must be a string or a real number, not 'NoneType'
[ERROR] The following algorithm call failed: cd "/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example" ;  /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 13086944 -bootstrap 'False' -criterion 'gini' -max_depth '47' -max_depth_type 'int' -max_features_special 'log2' -max_features_type 'special' -max_leaf_nodes '869' -max_leaf_nodes_type 'int' -min_impurity_decrease '1.7492582110184562E-4' -min_samples_leaf '52' -min_samples_split '32' -min_weight_fraction_leaf '0.10583492471874945' -n_estimators '324'  
[INFO ] 
=======================================================================================
SMAC has finished. Reason: wall-clock time limit (603.561 s) has been reached.   
Total number of runs performed: 81, total configurations tried: 28.
Total CPU time used: 62 s, total wallclock time used: 604 s.
SMAC's final incumbent: config 1 (internal ID: 0x000B), with estimated mean quality: -0.9630952380952381, based on 28 run(s) on 1 training instance(s).
Sample call for this final incumbent:
cd /rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example; /rwthfs/rz/cluster/home/vx475510/miniforge3/envs/sparkle/lib/python3.10/site-packages/sparkle/Components/smac2-v2.10.03-master-778/smac2_target_algorithm.py Solvers/RandomForest Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/tmp accuracy:max Instances/Iris/Iris.csv 0 60.0 2147483647 5505841 -bootstrap 'True' -criterion 'gini' -max_depth_type 'None' -max_features_special 'sqrt' -max_features_type 'special' -max_leaf_nodes_type 'None' -max_samples_type 'None' -min_impurity_decrease '0.0' -min_samples_leaf '1' -min_samples_split '2' -min_weight_fraction_leaf '0.0' -n_estimators '100' -oob_score 'True'
=======================================================================================
[INFO ] 
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Additional information about run 10 in:/rwthfs/rz/cluster/home/vx475510/satellite-operations-planning/sparkle_example/Output/Configuration/Raw_Data/SMAC2/RandomForest_Iris/outdir_train_configuration/RandomForest_Iris_scenario
----------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Organising done! See Output/Performance_Data/performance_data.csv for results.
